app/assets/stylesheets/govwales/components/_results.scss

```diff
@@ -0,0 +1,8 @@
+.app-c-actions-group {
+  .govuk-grid-column-one-third {
+    @media (min-width: 40.0625em) {
```

Comment:
Should we be using the Bootstrap variables for media queries?

Reply:
This is based on the GDS em breakpoints. I could change it to ours.

Reply:
Changed.